python-sdk/actionml/__init__.py

```diff
@@ -109,8 +109,8 @@ def __init__(self, url, threads=1, qsize=0, timeout=5):
             qsize=self.qsize,
             https=self.https,
             timeout=self.timeout,
-            client_id="Aladdin",
-            client_secret="OpenSesame"
```

Comment:
Oh, I liked Aladdin and OpenSesame. Where did you pick that up?

Reply:
wikipedia, basic auth example )